R/paretoObjectivesVaryWeights.R

```diff
@@ -2,9 +2,8 @@ paretoObjectivesVaryWeights <- function(MODEL, objective = hydromad.getOption("o
   objective <- buildCachedObjectiveFun(objective, MODEL)
   switch(hydromad.getOption("parallel")[["paretoObjectivesVaryWeights"]],
     "clusterApply" = {
-      if (!require("parallel")) stop('package parallel is required for paretoObjectivesVaryWeights if hydromad.getOption("parallel")[["paretoObjectivesVaryWeights"]]=="clusterApply"')
-      clusterExport(cl, c("objective", "fitBy", "MODEL"), envir = environment())
```

Comment:
As discussed, let's make this a `requireNamespace` because I'm not sure whether I prefer the default error message or the custom one

Reply:
fixed in 60d1286